Solutions/CohesitySecurity/Tests/az.py

```diff
@@ -42,3 +42,22 @@ def has_dup_incidents(resource_group, workspace_name):
 def get_incident_ids(resource_group, workspace_name):
     result = subprocess.run(['az', 'sentinel', 'incident', 'list', '--resource-group', resource_group, '--workspace-name', workspace_name], stdout=subprocess.PIPE)
     return [(jsObj["id"].split("/")[-1], jsObj["description"].split("Helios ID: ")[-1]) for jsObj in json.loads(result.stdout)]
+
+
+def search_alert_id_in_incident(alert_id, resource_group, workspace_name):
+    response = subprocess.run(['az', 'sentinel', 'incident', 'list',
```

Comment:
Question: When this script is run, are the logs stored somewhere so that it's useful for debugging? Otherwise, it's difficult to debug.

Reply:
As long the response is printed, I'm good.